pkgs/by-name/qu/quilt/package.nix

```
@@ -42,6 +42,14 @@ stdenv.mkDerivation rec {
     unixtools.getopt
   ];
 
+  strictDeps = true;
+
+  configureFlags = [
+    # configure only looks in $PATH by default,
+    # which does not include buildInputs if strictDeps is true
+    "--with-perl=${lib.getExe perl}"
+  ];
```

```suggestion
  nativeBuildInputs = [ perl ];
```
I expect this to work as long `perl` isn't removed from buildInputs.

---

And even if it worked, the configure script would pick up the wrong Perl for the build platform.

---

Allright. Understood the issue. That's a runtime perl. I'm sorry for the trouble.